assets/generated/api.md

```diff
@@ -11,6 +11,14 @@
 
 {{GetText $m.Operation.Description}}
 
+```sh
+curl -X {{$m.Request.Verb}} \
+    {{$.Swagger.Host}}{{$m.Request.URI}} \
+    -H 'Authorization: Bearer {{GetText "USE_YOUR_TOKEN"}}' \
+    {{if $m.Request.Example}}-d '{{$m.Request.Example}}'
```

Comment:
I think this will be broken if there's no example, as you'll add a trailing escaped newline. Not a big deal, but it can be avoided.